WordPress/src/main/res/values/strings.xml

```diff
@@ -265,6 +265,9 @@
     <string name="post_waiting_for_connection_pending">Post will be submitted for review next time your device is online</string>
     <string name="post_waiting_for_connection_scheduled">Post will be scheduled next time your device is online</string>
     <string name="post_waiting_for_connection_private">Private post will be published next time your device is online</string>
+    <string name="post_waiting_for_connection_publish_cancel">Changes will not be published</string>
```

Comment:
I'm pretty sure we'll update those messages when @osullivanchris returns. I think `Scheduling canceled` is quite misleading when you edit a scheduled post. Since the cancel button doesn't cancel the scheduling but only uploading (making public) the latest changes to the post.
Eg.
1. Schedule a post in online
2. Turn on airplane mode
3. Edit the scheduled post
4. Click on the Cancel button -> you haven't canceled the scheduling of the post but just the changes you made in step 3

Reply:
Thanks for the ping! Is there still an open question here that we need to discuss?